includes/functions.php

```diff
@@ -553,17 +553,6 @@ function is_single_user() {
  * @return boolean True if the site supports the block editor, false otherwise.
  */
 function site_supports_blocks() {
-	if ( \version_compare( \get_bloginfo( 'version' ), '5.9', '<' ) ) {
-		return false;
-	}
-
-	if (
-		! \function_exists( 'register_block_type_from_metadata' ) ||
```

Comment:
let's keep that, so that it will still work with classicpress or the classic editor enabled.

Reply:
The functions exist with the classic editor enabled. Should ClassicPress be a consideration?